game-service/src/main/java/org/libertybikes/game/core/AIPlayer.java

```diff
@@ -0,0 +1,14 @@
+/**
+ *
+ */
+package org.libertybikes.game.core;
+
+public abstract class AIPlayer extends Player {
+
+    public AIPlayer(String id, String name, short playerNum) {
```

Comment:
The implementation of AI doesn't need to know most of the data about the `Player` class (e.g. id, name, color).  The only thing it really makes use of is `setDirection()` which could be handled by changing `broadCastBoard` to return a DIRECTION instead of allowing the AI to set it directly.